src/OsmMom.sol

```diff
@@ -73,25 +72,37 @@ contract OsmMom {
         }
     }
 
-    mapping (bytes32 => address) public osms;
-
-    constructor() public {
+    constructor(address vat_) {
+        vat = vat_;
         owner = msg.sender;
+        emit SetOwner(msg.sender);
     }
 
-    function setOsm(bytes32 ilk, address osm) external note onlyOwner {
-        osms[ilk] = osm;
-    }
-
-    function setOwner(address owner_) external note onlyOwner {
+    function setOwner(address owner_) external onlyOwner {
         owner = owner_;
+        emit SetOwner(owner_);
     }
 
-    function setAuthority(address authority_) external note onlyOwner {
+    function setAuthority(address authority_) external onlyOwner {
         authority = authority_;
+        emit SetAuthority(authority_);
```

Comment:
following the above, if there is consensus in following the `clipper-mom` event pattern, we can move this line before to also include `oldAuthority`